src/media/UAudioDecoder_FFmpeg.pas

```diff
@@ -1449,9 +1274,6 @@ function TAudioDecoder_FFmpeg.InitializeDecoder: boolean;
 begin
   //Log.LogStatus('InitializeDecoder', 'UAudioDecoder_FFmpeg');
   FFmpegCore := TMediaCore_FFmpeg.GetInstance();
-  {$IF LIBAVFORMAT_VERSION < 58027100}
-  av_register_all();
```

Comment:
Are you sure about av_register_all being optional in FFmpeg 4.0?

Reply:
av_register_all was deprecated in 4.0, and my understanding is that it's no longer required to call it in 4.0 and later. See [here](https://github.com/leandromoreira/ffmpeg-libav-tutorial/issues/29) for example.